src/components/tr/tr-styles.jsx

```diff
@@ -24,6 +24,7 @@ const normal = theme => {
       borderWidth: 1,
       borderColor: color.grayLight,
     },
+    borderBottom: { borderBottom: `2px solid ${color.grayDarker}` },
```

Comment:
is this being used? couldn't find it in the examples

Reply:
It could